docs/guide/ffi_overview.md

```
@@ -0,0 +1,1067 @@
+
+**string_destroy**
+```c
+void string_destroy(char* s);
+```
+
+**byte_vector_destroy**
+```c
+void byte_vector_destroy(ByteVector* bytes);
+```
+
+**tari_vector_destroy**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)